packages/astro/src/core/errors/errors-data.ts

```diff
@@ -454,26 +454,75 @@ See https://docs.astro.build/en/guides/server-side-rendering/ for more informati
+	 */
 	ImageMissingAlt: {
 		title: 'Missing alt property',
 		code: 3022,
 		message: 'The alt property is required.',
 		hint: "The `alt` property is important for the purpose of accessibility, without it users using screen readers or other assistive technologies won't be able to understand what your image is supposed to represent. See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-alt for more information.",
 	},
+	/**
+	 * @docs
+	 * @see
+	 * - [Image Service API](https://docs.astro.build/en/reference/image-service-reference/)
+	 * @description
+	 * There was an error while loading the configured image service. This can be caused by various factors, such as your image service not properly exporting a compatible object in its default export or the path being wrong.
+	 *
+	 * If you believe that your service is properly configured and this error is wrong, please [open an issue](https://astro.build/issues/)
+	 */
 	InvalidImageService: {
 		title: 'Error while loading image service',
 		code: 3023,
 		message:
 			'There was an error loading the configured image service. Please see the stack trace for more information.',
 	},
+	/**
+	 * @docs
+	 * @message
+	 * Missing width and height attributes for `IMAGE_URL`. When using remote images, both dimensions are always required in order to avoid CLS.
+	 * @see
+	 * - [Assets (Experimental)](https://docs.astro.build/en/guides/assets/)
+	 * - [Image component#width-and-height](https://docs.astro.build/en/guides/assets/#width-and-height)
+	 * @description
+	 * For remote images, `width` and `height` cannot be inferred from the original file. As such, in order to avoid CLS, those two properties are always required.
+	 *
+	 * If your image is inside your `src` folder, you probably meant to import it instead. See [the Imports guide for more information](https://docs.astro.build/en/guides/imports/#other-assets).
+	 */
 	MissingImageDimension: {
 		title: 'Missing image dimensions',
 		code: 3024,
-		message: (missingDimension: 'width' | 'height' | 'both') =>
-			`For remote images, ${
-				missingDimension === 'both' ? 'width and height are' : `${missingDimension} is`
-			} required.`,
+		message: (missingDimension: 'width' | 'height' | 'both', imageURL: string) =>
+			`Missing ${
+				missingDimension === 'both'
+					? 'width and height attributes'
+					: `${missingDimension} attribute`
+			} for ${imageURL}. When using remote images, both dimensions are always required in order to avoid CLS.`,
+		hint: 'If your image is inside your `src` folder, you probably meant to import it instead. See [the Imports guide for more information](https://docs.astro.build/en/guides/imports/#other-assets).',
 	},
+	/**
+	 * @docs
+	 * @description
+	 * The built-in image services only support optimizing a limited amount of formats at the current time.
+	 *
+	 * For unsupported formats such as SVGs and gifs, you may be able to use an `img` tag directly:
```

Comment:
```suggestion
	 * For unsupported formats such as SVGs and GIFs, you may be able to use an `img` tag directly:
```
seems to be all caps around the web